mix.exs

```diff
@@ -41,7 +41,10 @@ defmodule Thrift.Mixfile do
      package: package(),
 
      # Dialyzer
-     dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:ex_unit, :mix, :public_key, :ssl]],
+     dialyzer: [
+       plt_add_deps: :transitive,
+       plt_add_apps: [:mix],
```

Comment:
IIRC I had to add `:ex_unit`, `:public_key`, and `:ssl` because they weren't being pulled into the PLT.  Did that get fixed some other way?